src/components/AlbumsList.js

```diff
@@ -26,6 +26,8 @@ const AlbumsList = () => {
   const [searchTerm, setSearchTerm] = useState(''); // for search input filed value
   const [message, setMessage] = useState(''); // for not found message
   const [errorMessage, setErrorMessage] = useState(''); // for error message
+  const [wishListAlbums, setWishListAlbums] = useState();
```

Comment:
Do we need `wishlistAlbums` to be in the component state? I'm not seeing this value be used elsewhere in this component, other than being set from within `useEffect`

Reply:
Removed